api/app/utils/untappd.ts

```diff
@@ -1,8 +1,10 @@
 
 import tds = require('./tds-promises');
-import {TYPES} from 'tedious';
+import { TYPES } from 'tedious';
 import request_promise = require('request-promise');
-import {Session} from '../models/Session';
+import { Activity } from '../models/Activity';
+import moment = require('moment');
```

Comment:
Did you end up using either of the moment libs? If not, consider removing references here & packages.json